packages/framework/src/Resources/views/Admin/Content/BestsellingProduct/list.html.twig

```diff
@@ -9,32 +9,32 @@
     
     {{ render(controller('ShopsysFrameworkBundle:Admin/Domain:domainTabs')) }}
 
-    {% macro categoryTreeItem(categoryDetails, selectedDomainId, bestsellingProductsInCategories) %}
+    {% macro categoryTreeItem(categoriesWithPreloadedChildren, selectedDomainId, bestsellingProductsInCategories) %}
```

Comment:
Future generations will thank you for this Rostislav. CategoryDetails vs CategoriesWithPreloadedChildren - the difference in the content description that results from the name of the variable is huge.